util/reggen/multi_register.py

```diff
@@ -169,3 +207,39 @@ def _asdict(self) -> Dict[str, object]:
         rd['compact'] = str(self.compact)
 
         return {'multireg': rd}
+
+    def apply_alias(self, alias_reg: object, where: str) -> None:
```

Comment:
As before, maybe worth strengthening this to make `alias_reg` of type `MultiRegister` (allowing you to drop the `isinstance` check below)

Reply:
Yes I went through all these `apply_alias` functions and strengthened the checks.